tests/installation/test_chef.py

```diff
@@ -1,4 +1,5 @@
 from packaging.tags import Tag
+
```

Comment:
Is this a related change? Same as the ones below.

Reply:
I didn't introduce these changes intentional. This is what `isort` has done. Don't know why.

Reply:
Weird. I guess isort ran outside of pre-commit.

Reply:
hm, yes. I'm used to run `isort` manually. Now I've run it via pre-commit and at least the tests passed here. But I have the strong feeling, that the pre-commit version doesn't respect the configuration in the `pyproject.toml`. Can you confirm this?

Reply:
Hmm, it might be that we never ran `isort` on the entire code base after the config was changed. It might be a good idea to do a PR with `isort` run on the whole codebase.